app/styles/design-system/components/button.css

```
@@ -0,0 +1,152 @@
+    .button {
+        --button-font-size: 1rem;
+        --button-margin: var(--cc-spacing-sm);
+        --button-padding-y: var(--cc-spacing-sm);
+        --button-padding-x: var(--cc-spacing-xs);
+
+        /* WCAG Target Size Requirements */
+        /* AA: 24px x 24px minimum, AAA: 44px x 44px minimum */
+        min-width: 44px;
+        min-height: 44px;
+
+        display: grid;
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)